### DISCUSSION TITLE
How to obtain the uploaded information such as url?

### DISCUSSION THREAD

**Discussion by fd6130** [async-aws/aws#1208] [Q&A]
After i upload an object to S3, the response give me

```
{
	"bucketKeyEnabled": null,
	"checksumCrc32": null,
	"checksumCrc32C": null,
	"checksumSha1": null,
	"checksumSha256": null,
	"etag": "\"0cbc6611f5540bd0809a388dc95a615b\"",
	"expiration": null,
	"requestCharged": null,
	"serverSideEncryption": "AES256",
	"sseCustomerAlgorithm": null,
	"sseCustomerKeyMd5": null,
	"sseKmsEncryptionContext": null,
	"sseKmsKeyId": null,
	"versionId": null
}
```

How can i obtain the uploaded object information such as url for frontend to render/access the resource?

**Comment by jderusse** [comment]
The SimpleS3 integration provides an easy way to get the URL

see https://async-aws.com/integration/simple-s3.html

note: Don't forget that by default, objects are privates, and URLs are signed with temporary token.

**Reply by fd6130** [reply]
Hi, thanks for the quick reply. By providing ACL "public-read" in the request body will make the uploaded object public right?

**Reply by jderusse** [reply]
never tried, but I believe that this should work
